docs/Initial-OnPrem-Setup/Setting-up-your-On-Premise-Machine.rst

```diff
@@ -0,0 +1,148 @@
+Setting up your On-Premise Machine
+==================================
+
+This tutorial is setting up a single node cluster (i.e. running FPGA bitstream builds and simulations on a single machine) for FireSim use.
+This single machine will serve as the "Manager Machine" that acts as a "head" node that all work will be completed on.
+``sudo`` access is not needed for the machine being setup except for when the U250 FPGA and corresponding software is installed.
+
+Xilinx Alveo FPGA Software Installation and Validation
+------------------------------------------------------
+
+On-Premise FPGA support currently only supports Xilinx Alveo U250 FPGAs (further referred to as a U250 FPGA).
+
+Requirements
+~~~~~~~~~~~~
+
+We require the following programs/packages installed from the Xilinx website in addition to a physical U250 installation:
+
+* Xilinx Vitis 2022.1
+* Xilinx XRT (corresponding with Vitis 2022.1)
+* Xilinx U250 board package (corresponding with Vitis 2022.1)
```

Comment:
We need to have direct links for how to do each of these. Especially the last one. IIRC the old page had some of these details.